data/categories.txt

```diff
@@ -91,43 +91,43 @@ fa:غذا|کجا غذا بخوریم
 
 # First keyword should match [food] definition in strings.txt!
 @food
-en:4Groceries|Food|Shop
-ru:4Продукты|Еда|4Магазин
-be:4Прадукты|Ежа|4Магазін
-bg:4Продукти|Храна|4Магазин
-ar:طعام|متجر|بقالية
-cs:Potraviny|Jídlo|Obchod
-da:Dagligvarer|Mad|Butik
-nl:Boodschappen|Kruidenierswinkels|Eten|Winkel
-fi:Elintarvikkeet|Ruoka|Kauppa
-fr:4Epiceries|Nourriture|Magasin
-de:4Lebensmittel|Essen|Geschäft|Laden
-hu:Élelmiszer|Ennivaló|Bolt|Üzlet
+en:4Groceries|Food
+ru:4Продукты|Еда
+ar:طعام|بقالية
+be:4Прадукты|Ежа
+bg:4Продукти|Храна
+cs:Potraviny|Jídlo
+da:Dagligvarer|Mad
+de:4Lebensmittel|Essen
+el:Παντοπωλεία|Φαγητό
+es:Alimentación|Productos|Comer
+et:Toidukaubad
+eu:Produktuak|Jan
+fa:غذا|فروشگاه|عطاری
+fi:Elintarvikkeet|Ruoka
+fr:4Epiceries|Nourriture
+he:חניה
+hu:Élelmiszer|Ennivaló|Üzlet
 id:Toserba|Makanan|Toko
 it:4Alimentari|Cibo|Negozio
-ja:食料雑貨|飲食|買い物
+ja:食料雑貨|飲食
 ko:식료품들|음식|쇼핑
 mr:किराणा|मंडई|ग्रोसरी
-nb:Dagligvarer|Mat|Butikk
-pl:Produkty|Jedzenie|Sklep
-pt:Mercearias|Comida|Alimentos|Loja
-pt-BR:Mercados|Alimentação|Loja
-ro:4Alimentare|Produse|Alimentație|Mâncare|Magazin
-es:Alimentación|Productos|Comer|Tienda
-et:Toidukaubad|Pood
-eu:Produktuak|Jan|Denda
-sv:Produkter|Mat|Butik|Affär
-th:ซื้อของกินของใช้|อาหาร|ร้านค้า
+nb:Dagligvarer|Mat
+nl:Boodschappen|Kruidenierswinkels|Eten
```

Comment:
nl:Boodschappen|Kruidenier|Eten|Supermarkt

Reply:
Well, only food (literally) should be here.  "Supermarkt" better to put into shop-supermarket (maybe convenience), right?